Documentation/platforms/arm64/qemu/boards/qemu-armv8a/index.rst

```diff
@@ -299,6 +299,70 @@ Running with QEMU + hvf on M1/MacBook Pro (macOS 12.6.1)
      -mon chardev=con,mode=readline -kernel ./nuttx
 
 ----------------------------------
+Single Core /w Xedge
+----------------------------------
+
+Configuring NuttX and compile:
```

Comment:
@JorgeGzm Can we add notes here that up to date GCC AARCH64 (i.e. 14.2) needs to be used as build may not work with older compilers (i.e. 11.3), see https://github.com/apache/nuttx-apps/pull/3120#issuecomment-3036348494 and https://github.com/apache/nuttx-apps/pull/3120#issuecomment-3036400714 :-)